tfhe/docs/guides/run_on_gpu.md

```diff
@@ -181,6 +181,7 @@ they are available and have peer access to GPU 0 via NVLink. To keep the API as
 ## Benchmarks
 
 All GPU benchmarks presented here were obtained on H100 GPUs, and rely on the multithreaded PBS algorithm. The cryptographic parameters `PARAM_GPU_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_3_KS_PBS` were used.
+When measuring GPU times on your own, set the environment variable `CUDA_MODULE_LOADING=EAGER` to avoid CUDA API overheads during the first kernel execution. 
```

Comment:
Maybe I would write this in a "Warning" section, like:

```
### Warning
When measuring GPU times on your own on Linux, set the environment variable `CUDA_MODULE_LOADING=EAGER` to avoid CUDA API overheads during the first kernel execution.